src/patternfly/components/InputGroup/examples/InputGroup.md

```diff
@@ -91,15 +83,15 @@ cssPrefix: pf-c-input-group
 {{/input-group}}
 <br>
 {{#> input-group}}
-  {{#> form-control controlType="input" input="true" form-control--attribute='type="search" id="textInput12" name="textInput12" aria-label="Input example with popover"'}}
+  {{#> form-control controlType="input" input="true" form-control--attribute='type="text" id="textInput12" name="textInput12" aria-label="Input example with popover"'}}
   {{/form-control}}
   {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Popover for input"'}}
     <i class="fas fa-question-circle" aria-hidden="true"></i>
   {{/button}}
 {{/input-group}}
 <br>
 {{#> input-group}}
-  {{#> form-control controlType="input" input="true" form-control--attribute='type="search" id="textInput14" name="textInput14" aria-label="Input example with plain unit"'}}
+  {{#> form-control controlType="input" input="true" form-control--attribute='type="number" id="textInput14" name="textInput14" aria-label="Input example with plain unit"'}}
   {{/form-control}}
```

Comment:
This is unrelated, but I noticed the a11y table is messed up. Not sure if that's an easy fix we could sneak in?
<img width="904" alt="Screen Shot 2022-02-24 at 7 23 49 PM" src="https://user-images.githubusercontent.com/35148959/155635963-de8a7817-bb31-4830-a52c-fffdfd798683.png">